amr-wind/equation_systems/temperature/source_terms/ABLMesoForcingTemp.cpp

```diff
@@ -183,9 +183,13 @@ amrex::Real ABLMesoForcingTemp::mean_temperature_heights(
 
     amrex::Vector<amrex::Real> error_T(m_nht);
 
+    amrex::Vector<amrex::Real> meso_ht(num_meso_ht);
+    amrex::Gpu::copy(
+        amrex::Gpu::deviceToHost, m_meso_ht.begin(), m_meso_ht.end(),
+        meso_ht.begin());
     for (int i = 0; i < m_nht; i++) {
         const amrex::Real height_interpolated_theta = amr_wind::interp::linear(
-            m_meso_ht, time_interpolated_theta, tavg.line_centroids()[i]);
+            meso_ht, time_interpolated_theta, tavg.line_centroids()[i]);
```

Comment:
Does this need a `tavg_lc` similar to change for velocity?

Reply:
I think it is mostly a no-op. I just did that for vel because it was happening twice...